### ISSUE TITLE
Spell Check Once

### ISSUE THREAD

**Issue by julesgilson** [streetsidesoftware/vscode-spell-checker#1117]
Maybe I am missing something....

Firstly, this extension works very well, but are the only options on or off? I don't really want it to always spell check - is it not possible to have it so we can run a command to run spell check on the open document - but not do it automatically? Toggling on then off again seems a bit clunky - just a "Spell Check Current Document Now" command would be great.

**Comment by hgsantana** [comment]
Please, this ^

I didn't find any good spell checker that does exactly that. Thank you for the awesome job with this spell checker (the best IMO).
I would love to have a discrete button in the status bar with a check on it, we press it and it runs for the current document alone.

**Comment by robross0606** [comment]
Would love same, as well as the option to spell check the whole project from a command and list them in Problems area.

**Comment by Jason3S** [comment]
@robross0606,

I think it is easiest to use the command line tool for that.

Open a VS Code Terminal window:

```
npx cspell --no-progress "**"
```

It doesn't show up in the problem window, but you can `ctrl-click` on the file name to jump to the file. 

![image](https://user-images.githubusercontent.com/3740137/142394653-3f462cfe-e4ec-4711-9c68-7f5eecb862e3.png)

**Comment by robross0606** [comment]
Yes, I found cspell after making this comment.  But that tool being available doesn’t quite negate the request for it being integrated into the vscode plugin.

**Comment by SRNissen** [comment]
I found this issue doing a search to make sure I wasn't duplicating.

I have an enhancement suggestion that is approximately similar:

I occasionally want spell checking of *some* words, but (for various reasons) a persistent-on spell checker is not desired.

Creating a "spell check once" command would solve 100% of my problem but what I *really* want is the ability to right-click a word and ask for spell checking of that specific word.
